### PR TITLE
Document tidyNode::getNextSibling() and tidyNode::getPreviousSibling()

### PR DESCRIPTION
**Pull request by nielsdos**
Drive by fix in second commit too

### CODE REVIEW

File: reference/tidy/tidynode/getnextsibling.xml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+   Returns a <type>tidyNode</type> if the node has a next sibling, or &null;
+   otherwise.
+  </para>
+ </refsect1>
+
+ <refsect1 role="examples">
+  &reftitle.examples;
+  <para>
+   <example>
+    <title><function>tidyNode::getNextSibling</function> example</title>
```
Comment (Girgias) [inline reference/tidy/tidynode/getnextsibling.xml:37]: I know a lot of the documentation does this for reasons I don't know, but the wrapping `<para>` tag around `<example>` is useless.

File: reference/tidy/tidynode/getnextsibling.xml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="tidyNode">
+   <modifier>public</modifier> <type class="union"><type>tidyNode</type><type>null</type></type><methodname>tidyNode::getNextSibling</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
+   Returns the next sibling node of the current node.
+  </para>
```
Comment (Girgias) [inline reference/tidy/tidynode/getnextsibling.xml:17]: Could be a `<simpara>`

**Comment by nielsdos** [comment]
I hope I did it right. It didn't generate the links locally for the "see also" section, but the existing "see also" sections also didn't have links locally so... :shrug:

File: reference/tidy/tidynode/getnextsibling.xml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+ </refsect1>
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
```
Comment (Girgias) [inline reference/tidy/tidynode/getnextsibling.xml:27]: Ditto could be simpara

File: reference/tidy/tidynode/getprevioussibling.xml [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  <refpurpose>Returns the previous sibling node of the current node</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="tidyNode">
+   <modifier>public</modifier> <type class="union"><type>tidyNode</type><type>null</type></type><methodname>tidyNode::getPreviousSibling</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
```
Comment (Girgias) [inline reference/tidy/tidynode/getprevioussibling.xml:15]: Ditto simpara

File: reference/tidy/tidynode/getprevioussibling.xml [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+ </refsect1>
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
```
Comment (Girgias) [inline reference/tidy/tidynode/getprevioussibling.xml:27]: Ditto simpara

File: reference/tidy/tidynode/getprevioussibling.xml [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+   Returns a <type>tidyNode</type> if the node has a previous sibling, or &null;
+   otherwise.
+  </para>
+ </refsect1>
+
+ <refsect1 role="examples">
+  &reftitle.examples;
+  <para>
+   <example>
```
Comment (Girgias) [inline reference/tidy/tidynode/getprevioussibling.xml:36]: Ditto useless para wrapping tag

**Review by Girgias — APPROVED** [review]
Minor markup nits, but LGTM. Thank you!

**Pull request merged**